git/checkout.go

```diff
@@ -22,10 +22,8 @@ type CheckoutOptions struct {
 	// Not implemented
 	Stage Stage
 
-	// Not implemented
-	Branch string // -b
-	// Not implemented
-	ForceBranch bool // use branch as -B
+	Branch      string // -b
+	ForceBranch bool   // use branch as -B
```

Comment:
I don't see where ForceBranch is being used. Did you want to preserve the // Not implemented comment for it until it is used?

Reply:
Sure, I added the comment back.